contracts/ronin/validator/SlashingExecution.sol

```diff
@@ -34,8 +34,7 @@ abstract contract SlashingExecution is
     _blockProducerJailedBlock[_validatorAddr] = Math.max(_newJailedUntil, _blockProducerJailedBlock[_validatorAddr]);
 
     if (_slashAmount > 0) {
-      uint256 _actualAmount = _stakingContract.execDeductStakingAmount(_validatorAddr, _slashAmount);
-      _totalDeprecatedReward += _actualAmount;
+      _totalDeprecatedReward += _stakingContract.execDeductStakingAmount(_validatorAddr, _slashAmount);
```

Comment:
The current code reserve the `_actualAmount` for readability, to infer the actual amount of deduction when a validator gets slashed. Please add a comment to note what is happening here if you need to shorten these lines.

Reply:
commented.